lib/pluginUtils.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const assert = require('assert')
+
+function getMeta (fn) {
+  return fn[Symbol.for('plugin-meta')]
```

Comment:
maybe we should validate that if we have a `plugin-meta`, it has a name? Should we check the format of this with a json schema?